src/packages/excalidraw/example/App.tsx

```diff
@@ -510,7 +510,7 @@ export default function App() {
   return (
     <div className="App" ref={appRef}>
       <h1> Excalidraw Example</h1>
-      <Sidebar>
+      <ExampleSidebar>
```

Comment:
can you update the Sidebar example to use `Sidebar.Header` as well ?
Also better to rename to `SidebarExample` instead

Reply:
> can you update the Sidebar example to use Sidebar.Header as well ?

This is a different sidebar, the one you added to demonstrate offsetting: 

![image](https://user-images.githubusercontent.com/5153846/188464601-aecdbcc0-7c5a-4c91-922c-42882a79fc4c.png)

> Also better to rename to SidebarExample instead

I want to avoid it being sorted on top when the importing/opening files.

Reply:
> This is a different sidebar, the one you added to demonstrate offsetting:

True but we can reuse the sidebar here as well right ?

>can you update the Sidebar example to use Sidebar.Header as well ?

This is a different sidebar, the one you added to demonstrate offsetting:


> I want to avoid it being sorted on top when the importing/opening files.

Well we don't need the word `example` at all since its inside example folder so lets keep it `Sidebar.jsx`, `Sidebar.scss` instead

Reply:
> True but we can reuse the sidebar here as well right ?

Unclear what you mean by reuse. I've meant to add an example Sidebar to the example app, but that's different from what we're discussing here.

> Well we don't need the word example at all since its inside example folder so lets keep it Sidebar.jsx, Sidebar.scss instead

We do if we want to distinguish an example file from a regular component. When `Cmd-P` for a file and type `Sidebar`, I don't want to be deciphering from the filepath which component is the real one.

We should consider moving the example app outside of our repo into an example repo.

Reply:
> Unclear what you mean by reuse. I've meant to add an example Sidebar to the example app, but that's different from what we're discussing here.

I mean lets use the new side bar component instead of the one which is there in the example as well as `Sidebar`  will now be exported from the package as well

> We do if we want to distinguish an example file from a regular component. When Cmd-P for a file and type Sidebar, I don't want to be deciphering from the filepath which component is the real one.

Well in that case we should follow the same for `index.tsx` files as well, having multiple `index.tsx` it often is confusing which one I am looking for when searching for `index` file

> We should consider moving the example app outside of our repo into an example repo
The `example` is in the same repo so that its easier to test stuff when making changes in the package. Once we move to `monorepo` it should be fine.

Reply:
> I mean lets use the new side bar component instead of the one which is there in the example as well as Sidebar will now be exported from the package as well

Done.

> Well in that case we should follow the same for index.tsx files as well, having multiple index.tsx it often is confusing which one I am looking for when searching for index file

Agreed. For some other time though.